.github/workflows/benchmark.yml

```diff
@@ -1,11 +1,17 @@
 name: Benchmark PR vs main
 on:
   workflow_dispatch:
+  pull_request_review:
+    types: [submitted]
   pull_request:
     branches: [ main ]
-  
+    types: [synchronize]
+    paths:
+      - '**/*.swift'
```

Comment:
add the CI file itself as well?

Reply:
Mhh why?

Reply:
If we change the CI file we'd want to make sure it is still functional?

Reply:
Yes this. We can also ignore the tests as well